eberron.md

```diff
@@ -407,7 +449,10 @@ You have trained your body and mind for battle, learning to ignore the pain and
 
 While in the berserker form:
 - You cannot cast spells
-- Re-roll 1 wound die for each of your attacks
+- You add 1 wound die to attacks and maneuvers
+- You must attack each round if able, using your move and sprint actions if no creatures are nearby
+
+You may attempt to end your berserker rage once each turn, making a Willpower save against 2 bad dice, or 4 bad dice if you have taken damage since your last turn. On success the rage ends. You add a superior die for every turn you end without attacking.
```

Comment:
adding a conditional to the willpower save, easier to save if nothing is actively hurting you, harder to save if your in combat still?